lib/internal/util.js

```diff
@@ -366,8 +366,20 @@ function isInsideNodeModules() {
   return false;
 }
 
+const kCustomFeatureDetectionSymbol = Symbol('node.util.features');
```

Comment:
I think this should be a `Symbol.for('node.util.features')`. It makes things a lot simpler for people maintaining isomorphic modules.